EVENT.md

```diff
@@ -603,6 +604,65 @@ Set-Cookie: ar_debug=1; SameSite=None; Secure; HttpOnly
 ```
 If a cookie of this form is not present, debugging information will be ignored.
 
+### Optional: error reports
+
+We also introduce a non-cookie-based debugging framework to allow developers to
+monitor certain failures in the attribution registrations.
+
+The browser will send error reports in the following source-registration failure
+modes:
+
+* a source is rejected due to the [reporting origin
+  limits](#reporting-origin-limits)
+* a source is rejected due to the [destination
+  limits](#limiting-the-number-of-unique-destinations-covered-by-pending-sources)
+
+The browser will send non-credentialed secure HTTP `POST` requests to the
+reporting endpoints, see [below](#reporting-endpoints). The report data is
+included in the request body as a JSON object:
+```jsonc
+{
+  "type": "<report type>", // could be "unattributed-reporting-origin-limit" or "source-destination-limit"
+  "body": {
+    "limit": 100, // the browser's limit
+    "source_event_id": "<source event id in the source registration>",
+    "source_site": "https://source.example",
+    "destination_site": "https://destination.example",
+    "reporting_origin": "https://reporting.example.com"
+  }
+}
+```
+
+These error reports will be sent immediately upon the error occuring during
+source registration.
+
+Note that if other failures are reported using this framework, the browser may
+enforce a random small delay and omit some data (e.g. `source_event_id`) to
+preserve privacy.
+
+#### Reporting endpoints
+
+The reporting origins may opt in to receiving error reports by adding a new
+`error_reporting` dictionary field to the `Attribution-Reporting-Register-Source` header:
+```jsonc
+{
+  ... // existing fields
+
+  "error_reporting": true // defaults to false if not present
+}
+```
+
+The error reports will be sent to a new endpoint:
+```
+https://<reporting origin>/.well-known/attribution-reporting/error
+```
+
+The top-level sites may opt in to receiving error reports by declaring a reporting
+endpoint via an HTTP header in the top-level document response:
+```
+Attribution-Reporting-Error-Endpoint: https://example.com/reports // an arbitrary URL
```

Comment:
To avoid confusion, I'd replace the `// an arbitrary URL` comment with a new paragraph beneath the code block, and probably also mention that while the URL can be arbitrary, its origin must be potentially trustworthy.

Reply:
Sure, that's fine.